XamarinCommunityToolkit/Actions/Snackbar/Snackbar.android.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using Xamarin.Forms;
+using Android.Widget;
+using Xamarin.Forms.Platform.Android;
+using Xamarin.CommunityToolkit.Extensions;
+#if MONOANDROID10_0
+using AppCompatAlertDialog = AndroidX.AppCompat.App.AlertDialog;
+using AppCompatActivity = AndroidX.AppCompat.App.AppCompatActivity;
+using AndroidSnackbar = Google.Android.Material.Snackbar.Snackbar;
+#else
+using AppCompatAlertDialog = global::Android.Support.V7.App.AlertDialog;
+using AppCompatActivity = global::Android.Support.V7.App.AppCompatActivity;
+using AndroidSnackbar = global::Android.Support.Design.Widget.Snackbar;
+#endif
+
+namespace Xamarin.CommunityToolkit.Actions.Snackbar
+{
+	using global::Android.App;
```

Comment:
just a question: is there any purpose why we put this using inside the namespace?

Reply:
sorry, my default VS settings